### PR TITLE
DEVAI-134: Remove unused value references

### PR DESCRIPTION
**Pull request by Jdubrick**
### What does this PR do?:

This PR removes unused values from the template. These values were initially used at some point and we moved away from them without removing them from the template. This overall should help reduce any confusion/clutter in our templates.

### Which issue(s) this PR fixes:

Issue: https://issues.redhat.com/browse/DEVAI-134

### PR acceptance criteria:
Testing and documentation do not need to be complete in order for this PR to be approved. We just need to ensure tracking issues are opened and linked to this PR, if they are not in the PR scope due to various constraints.

- [x] Tested and Verified

  

- [ ] Documentation (READMEs, Product Docs, Blogs, Education Modules, etc.)

   

### How to test changes / Special notes to the reviewer:

I installed RHDH on my ROSA cluster (using RHTAP installer) and then deployed every template and verified they worked for sanity. If you would like to do the same that is all you'd have to do.

### CODE REVIEW

**Comment by Jdubrick** [comment]
FYI the `generate.sh` warning is pulling changes in a draft PR to the template repo, I ran `./generate.sh` to incorporate my changes across the templates.

**Review by thepetk — DISMISSED** [review]
Nice cleanup :)

lgtm, I'd suggest waiting for https://github.com/redhat-ai-dev/ai-lab-template/pull/40 to be merged if possible.

File: properties [hunk 1]
```diff
@@ -1,6 +1,4 @@
 # PE can replace these values with on-prem host
-export GITHUB__DEFAULT__HOST=github.com
```
Comment (yangcao77) [inline properties:2]: this should be referenced in https://github.com/redhat-ai-dev/ai-lab-template/blob/56d58e8e9a85c47f0094c92c3112bf9243c7e3ea/skeleton/template.yaml#L172, as `${GITHUB__DEFAULT__HOST}`, same for the gitlab case

on-prem host should be supported. see RHTAP customizing template doc as example: 
https://gitlab.cee.redhat.com/software-supply-chain-documentation/rhtap/-/blob/main/topics/proc_customizing-sample-software-templates.adoc?ref_type=heads

File: templates/audio-to-text/template.yaml [hunk 2]
```diff
@@ -244,15 +244,11 @@ spec:
           description: This is GitOps manifest for ${{ parameters.name }}
           namespace: ${{ parameters.namespace }}
           rhtapNamespace: rhtap
```
Comment (yangcao77) [inline templates/audio-to-text/template.yaml:246]: asides this cleanup pr, but we should rename those params mentions `rhtap`

Comment (Jdubrick) [reply templates/audio-to-text/template.yaml:246]: Yeah I forgot to rename the vars when I opened https://github.com/redhat-ai-dev/ai-lab-template/pull/42, so the values are not `rhtap` anymore but I will include the renaming as part of this PR!

**Review by maysunfaisal — APPROVED** [review]
lgtm

**Pull request merged**